l3.S

```diff
@@ -2,27 +2,31 @@
 .extern l3_log
 
 l3__log_fast:
-    mov %fs:0xfffffffffffffffc,%eax
+    mov %fs:0xfffffffffffffffc,%eax // did we already stuff the tid in TLS?
```

Comment:
Most comments are declarative, but this one is asking a question.

I think it may be simpler, and easier to follow, if this comment is also phrased like it's on L8 for the other `mov` command.

Truly, I don't even understand what this assembly does -- so will leave it up to you to rephrase this as a statement rather than a question.